primitives/rpc/src/lib.rs

```diff
@@ -96,6 +96,8 @@ sp_api::decl_runtime_apis! {
 			Option<Vec<ethereum::Receipt>>,
 			Option<Vec<TransactionStatus>>
 		);
+		/// Return the current block gas limit
+		fn current_block_gas_limit() -> U256;
```

Comment:
Why can't we just use `current_block`?

Reply:
The reason why I think using `current_block` is better is that this also affects our RPC storage access trait.

Reply:
Good catch, this wasn't supposed to be in this PR anyway. I've reverted this in 1f8de10.

I will consider PRing it individually and take your feedback into account.